modules/distr-tracing-tempo-install-web-console.adoc

```diff
@@ -14,7 +14,12 @@ You can install the {TempoShortName} from the *Administrator* view of the web co
 
 * For {product-dedicated}, you must be logged in using an account with the `dedicated-admin` role.
 
-* You are using a supported provider of object storage: link:https://www.redhat.com/en/technologies/cloud-computing/openshift-data-foundation[Red Hat OpenShift Data Foundation], link:https://min.io/[MinIO], link:https://aws.amazon.com/s3/[Amazon S3], link:https://azure.microsoft.com/en-us/products/storage/blobs/[Azure Blob Storage], link:https://cloud.google.com/storage/[Google Cloud Storage].
+* You have completed setting up the required object storage by a supported provider: link:https://www.redhat.com/en/technologies/cloud-computing/openshift-data-foundation[Red Hat OpenShift Data Foundation], link:https://min.io/[MinIO], link:https://aws.amazon.com/s3/[Amazon S3], link:https://azure.microsoft.com/en-us/products/storage/blobs/[Azure Blob Storage], link:https://cloud.google.com/storage/[Google Cloud Storage]. For more details, see _Object storage setup_.
```

Comment:
Thank you, I've applied your suggestion manually.

Reply:
I think the reader should see the referenced section because the required info is there. As for turning this into a bullet list, this would add 6+ more lines of text of one to a few words long, not sure if this is worth the real estate. Also, formatting it as a bullet list might backfire on readability: all of the other options become visible, which means readers could initially misread it as a list of required items rather than a list of alternatives.